packages/world/src/WorldFactory.sol

```diff
@@ -28,12 +25,14 @@ contract WorldFactory is IWorldFactory {
   /**
    * @notice Deploys a new World instance, installs the InitModule and transfers ownership to the caller.
    * @dev Uses the Create2 for deterministic deployment.
+   * @param _salt User defined salt for deterministic world addresses across chains
    * @return worldAddress The address of the newly deployed World contract.
    */
-  function deployWorld() public returns (address worldAddress) {
+  function deployWorld(bytes memory _salt) public returns (address worldAddress) {
```

Comment:
small API thing: because Solidity supports calling functions with named parameters now, it might make sense to call this `salt` and internally we can call the combined one `_salt`

Reply:
Fixed!